index.js

```diff
@@ -27,23 +26,23 @@ const currentTime = () => {
 
 // display books
 
-function displayBooks() {
+const displayBooks = () => {
   container.innerHTML = '';
   bookList.books.forEach((books, index) => {
     container.innerHTML += `<ul>
     <li>"${books.title}" by ${books.author}</li>
     <li><button class = "remove" onlick= "removebook()" data-index="${index}">Remove</button></li>
     </ul>`;
   });
-}
+};
```

Comment:
[OPTIONAL] Nice work overall 💯 but in order to improve it you could consider showing a message when there are no books 😉 this is what it shows 👇🏽 

<img width="1463" alt="Screenshot 2023-03-27 at 07 38 37" src="https://user-images.githubusercontent.com/45739963/227930926-6a29fa9e-a0ab-4ad6-90fd-fe05c6216e4b.png">